### DISCUSSION TITLE
How can I change the font of the system in Cosmos

### DISCUSSION THREAD

**Discussion by Dell-Optiplex-390** [CosmosOS/Cosmos#2082] [Q&A]
Hello, I'd like to change the terminals font since the original font the command prompt uses looks to old. Thanks!

**Comment by quajak** [comment]
You can use the VGATextMode.SetFont method https://github.com/CosmosOS/Cosmos/blob/fd799bc45902e65151ef1ef8ad601dab1a10c72e/source/Cosmos.System2/Graphics/VGATextMode.cs#L64

**Reply by Dell-Optiplex-390** [reply]
Right thanks! But what should there be in byte[]? The Base64 Code?

**Reply by quajak** [reply]
No, the format required is described here https://wiki.osdev.org/VGA_Fonts. Every character has a bitmap with one byte for each line and the characters bitmaps are written one after the other in fontData byte array.

You should be able to find example fonts on the internet if you search for VGA Font.

**Reply by Dell-Optiplex-390** [reply]
I'm a bit new to Cosmos so i'm not sure what some stuff means. Is there a YouTube Tutorial I can follow?

**Reply by quajak** [reply]
Not that I know of.  In the graphics test kernel there is an example where the font is changed.
